Gemfile

```diff
@@ -84,6 +84,8 @@ group :development, :test do
   gem 'guard-rspec', require: false
 
   gem 'factory_bot_rails'
+  # TODO: switch to a version number once that version is released
```

Comment:
I think you'd be better off with the released version.  This sha is from 2017.  Also changes to the gemfile should be done in a separate PR from the ruby upgrade.

Reply:
Please see https://github.com/department-of-veterans-affairs/gibct-data-service/pull/429 as the plan is to merge that PR before this one